examples/interactivity/multi-feature.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Multi feature | CARTO</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta charset="UTF-8">
+  <!-- Include CARTO GL JS -->
+  <script src="../../dist/carto-gl.js"></script>
+  <!-- Include Mapbox GL JS -->
+  <script src="../../vendor/mapbox-gl-dev.js"></script>
+  <!-- Include Mapbox GL CSS -->
+  <link href="../../vendor/mapbox-gl-dev.css" rel="stylesheet" />
+  <style>
+    * {
+      margin: 0;
+      padding: 0;
+      box-sizing: border-box;
+    }
+    #map {
+      position: absolute;
+      height: 100%;
+      width: 100%;
+      z-index: 0;
+    }
+    #controls {
+      position: absolute;
+      top: 12px;
+      right: 12px;
+      width: 200px;
+      z-index: 2;
+    }
+    #controls h2 {
+      font: 300 12px/16px 'Open Sans';
+      background: rgba(0, 0, 0, 0.64);
+      border-radius: 4px;
+      padding: 8px 12px;
+      color: #fff;
+    }
+    .container {
+      padding: 8px 12px;
+      background: white;
+      box-shadow: 0 0 16px rgba(0, 0, 0, 0.12);
+      border-radius: 4px;
+      color: #2E3C43;
+      margin-top: 12px;
+    }
+    #content h3 {
+      font: 400 16px/22px 'Open Sans';
+      margin-bottom: 4px;
+    }
+    #content p {
+      font: 400 12px/14px 'Open Sans';
+    }
+  </style>
+</head>
+<body>
+  <div id="map"></div>
+  <div id="controls">
+    <h2></h2>
+    <div id="content">
+    </div>
+  </div>
+  <script>
+    const map = new mapboxgl.Map({
+      container: 'map',
+      style: 'https://basemaps.cartocdn.com/gl/voyager-gl-style/style.json',
+      center: [0, 40],
+      zoom: 4,
+      dragRotate: false
+    });
+
+    carto.setDefaultAuth({
+      user: 'cartogl',
+      apiKey: 'YOUR_API_KEY'
+    });
+
+    const spainCitiesSource = new carto.source.SQL(`
+      SELECT *
+      FROM ne_10m_populated_places_simple
+      WHERE adm0name = \'Spain\'
+    `);
+    const spainCitiesStyle = new carto.Style(`
+      width: log($pop_max + 10),
+      color: opacity(ramp($name, bold), 0.7)
+      strokeWidth: 1
+      strokeColor: rgb(255, 255, 255)
+    `);
+    const spainCitiesLayer = new carto.Layer('spainCitiesLayer', spainCitiesSource, spainCitiesStyle);
+
+    const europeCountriesSource = new carto.source.Dataset(`
+      ne_adm0_europe
+    `);
+    const europeCountriesStyle = new carto.Style(`
+      color: ramp($admin, bold)
+    `);
+    const europeCountriesLayer = new carto.Layer('europeCountriesLayer', europeCountriesSource, europeCountriesStyle);
+
+    europeCountriesLayer.on('loaded', () => {
+      updateTitle('Click on the cities / countries');
+
+      // Detect first `europeCountriesLayer` features, then `spainCitiesLayer` features
+      const interactivity = new carto.Interactivity([europeCountriesLayer, spainCitiesLayer]);
```

Comment:
how do you know here that spainCitiesLayer has been loaded?

Reply:
Aha! In this case because the `spainCitiesLayer` is faster than the other layer.

I just found this "problem" while doing the example. We need to talk about it in the today's meeting.

Reply:
Yes! I made the same mistake the other week, it's clearly error-prone and part of the API. We should definitely talk about it.